jupyterlite_sphinx/jupyterlite_sphinx.js

```diff
@@ -39,3 +39,38 @@ window.jupyterliteConcatSearchParams = (iframeSrc, params) => {
     return iframeSrc;
   }
 }
+
+
+window.tryExamplesShowIframe = (
+    examplesContainerId, iframeContainerId, iframeParentContainerId, iframeSrc
+) => {
+    const examplesContainer = document.getElementById(examplesContainerId);
+    const iframeParentContainer = document.getElementById(iframeParentContainerId);
+    const iframeContainer = document.getElementById(iframeContainerId);
+
+    let iframe = iframeContainer.querySelector('iframe.jupyterlite_sphinx_raw_iframe');
+
+    if (!iframe) {
+	const examples = examplesContainer.querySelector('.try_examples_content');
+	iframe = document.createElement('iframe');
+	iframe.src = iframeSrc;
+	iframe.style.width = '100%';
+	iframe.style.height = `${examples.offsetHeight}px`;
+	iframe.classList.add('jupyterlite_sphinx_raw_iframe');
+	examplesContainer.classList.add("hidden");
+	iframeContainer.appendChild(iframe);
+    }
+    else {
+	examplesContainer.classList.add("hidden");
+    }
+    iframeParentContainer.classList.remove("hidden");
```

Comment:
Small indent nitpicking

```suggestion
    if (!iframe) {
	      const examples = examplesContainer.querySelector('.try_examples_content');
	      iframe = document.createElement('iframe');
	      iframe.src = iframeSrc;
	      iframe.style.width = '100%';
	      iframe.style.height = `${examples.offsetHeight}px`;
	      iframe.classList.add('jupyterlite_sphinx_raw_iframe');
	      examplesContainer.classList.add("hidden");
	      iframeContainer.appendChild(iframe);
    } else {
	      examplesContainer.classList.add("hidden");
    }
    iframeParentContainer.classList.remove("hidden");
```

Reply:
Thanks. New to Javascript so not really aware of the best ways to format things.